test/model/src/driver/kv_workload_runner_wt.cpp

```diff
@@ -517,6 +530,32 @@ kv_workload_runner_wt::do_operation(const operation::truncate &op)
       session->session(), table_uri(op.table_id), cursor1, cursor2, op.start, op.stop);
 }
 
+/*
+ * kv_workload_runner_wt::do_operation --
+ *     Execute the given workload operation in WiredTiger.
+ */
+int
+kv_workload_runner_wt::do_operation(const operation::wt_config &op)
+{
+    std::unique_lock lock(_connection_lock);
+
+    size_t l = op.value.size() + 1; /* In-memory size, including the NUL byte. */
+    const char *v = op.value.c_str();
+
+    if (op.type == "connection") {
+        if (l > sizeof(_state->connection_config))
+            throw model_exception("The connection config is too long");
+        memcpy(_state->connection_config, v, l);
+    } else if (op.type == "table") {
+        if (l > sizeof(_state->table_config))
+            throw model_exception("The connection config is too long");
```

Comment:
Change "connection" to "table".

Reply:
Nice catch. Thanks for reviewing this carefully! Fixed in [6ea43c7](https://github.com/wiredtiger/wiredtiger/pull/10943/commits/6ea43c73999e1936f25c20d81eb3f8004acd0a3a).